From: Ilya Dryomov <idryomov@gmail.com>
Date: Thu, 3 May 2018 16:26:55 +0200
Subject: ceph: show wsize only if non-default
Git-commit: 6dd4940ba5f96270ad428351cd88daf9ab871a97
Patch-mainline: v4.18-rc1
References: FATE#324714

This is how it was before commit 95cca2b44e54 ("ceph: limit osd write
size") went in.

Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: Luis Henriques <lhenriques@suse.com>
---
 fs/ceph/super.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/ceph/super.c b/fs/ceph/super.c
index b33082e6878f..3c1155803444 100644
--- a/fs/ceph/super.c
+++ b/fs/ceph/super.c
@@ -551,7 +551,7 @@ static int ceph_show_options(struct seq_file *m, struct dentry *root)
 
 	if (fsopt->mds_namespace)
 		seq_show_option(m, "mds_namespace", fsopt->mds_namespace);
-	if (fsopt->wsize)
+	if (fsopt->wsize != CEPH_MAX_WRITE_SIZE)
 		seq_printf(m, ",wsize=%d", fsopt->wsize);
 	if (fsopt->rsize != CEPH_MAX_READ_SIZE)
 		seq_printf(m, ",rsize=%d", fsopt->rsize);

